integration-test/Css.ts

```diff
@@ -338,6 +338,14 @@ class CssBuilder<T extends Properties1> {
   get f12() { return this.add("fontSize", "12px"); }
   get f10() { return this.add("fontSize", "10px"); }
 
+  // typographyRules
+  get measure() { return this.add("maxWidth", "30em"); }
```

Comment:
Oh cool, I'd read about the 30em rule of thumb width before, as just a general publishing / book layout type guideline; didn't realize that tachyons had it. Neat!